styles/global.scss

```diff
@@ -208,6 +208,12 @@ button.button.is-bordered {
   }
 }
 
+.buttons {
+  .b-tooltip {
+    width: 100%;
+  }
+}
```

Comment:
Why would you want to make it global? I mean that if any other component that have tooltip on buttons will now have a width of 100% and it could break something somewhere.
What did you try to fix here? Couldn't it be scoped to component?

Reply:
@roiLeo now I understood your point, I thought you want this to be global, my bad. I will add this to component scope.